src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -316,6 +321,57 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                 return MaybeInjectCoercion(node, new CallNode(context.GetIRContext(node), func, args), context);
             }
 
+            private IntermediateNode MaybeInjectPreProcessor(IntermediateNode argIR, IRPreProcessor preProcessor)
+            {
+                switch (preProcessor)
+                {
+                    case IRPreProcessor.BlankToZero:
+                        return BlankToZeroIRCallNode(argIR);
+                    case IRPreProcessor.BlankToEmptyString: 
+                        return BlankToEmptyStringIRCallNode(argIR);
+                    case IRPreProcessor.NumberTruncate:
+                        return NumberTruncateIRCallNode(argIR);
+                    default:
+                        return argIR;
```

Comment:
nit: 
safest is to handle none explicitly and then for "default" to throw unhandled exception.